src/sdl2_ui.cpp

```diff
@@ -301,7 +301,7 @@ bool Sdl2Ui::RefreshDisplayMode() {
 			return false;
 		}
 
-		auto renderer_sg = makeScopeGuard([&]() {
+		auto renderer_sg = lcf::makeScopeGuard([&]() {
```

Comment:
The lcf-namespace nicely indicates now that scope guard doesn't really fit there - A liblcf dependency in SDL2_Ui :thinking:

Reply:
Yep, it's ugly. But the alternative is another repo and then we have 3 to keep in sync 😭 .

Reply:
Also thought about having this in a git subtree. "inside, but outside"